zookeeper-api/src/test/java/org/apache/helix/zookeeper/impl/client/TestRawZkClient.java

```diff
@@ -146,7 +123,6 @@ void testCreatePersistentWithTTL() {
     // Clean up
     zkClient.deleteRecursively(parentPath);
     zkClient.close();
-    System.clearProperty("zookeeper.extendedTypesEnabled");
   }
 
   @Test
```

Comment:
When you said parallel test execution, do you mean test classes running in parallel or test cases (methods) running in parallel?
For former, I remember we had a setting that prevents classes from running in parallel; for latter, we could set test dependency to force sequential execution.

Reply:
I remember it was a setting in the test framework... Would you be able to try serializing test case executions using "dependsOn"? That alone should prevent parallel test issues.